### PR TITLE
Retry connecting to pg

### PR DESCRIPTION
**Pull request by ikitiki**
- add connect_timeout parameter to the pg connection string 
- if for some net-related reasons we failed to connect to the postgresql db, try again

### CODE REVIEW

File: pkg/cluster/pg.go [hunk 1]
```diff
@@ -45,25 +46,42 @@ func (c *Cluster) databaseAccessDisabled() bool {
...
 			if err2 := conn.Close(); err2 != nil {
 				c.logger.Errorf("error when closing PostgreSQL connection after another error: %v", err2)
+				return false, err2
 			}
-			return err
-		}
 
-		c.pgDb = conn
+			if _, ok := err.(*net.OpError); ok {
+				return false, nil
```
Comment (alexeyklyukin) [inline pkg/cluster/pg.go]: Why would we hide this error?

Comment (ikitiki) [reply pkg/cluster/pg.go]: you mean we should log it?

Comment (alexeyklyukin) [reply pkg/cluster/pg.go]: yes, logging it would help us in finding the root cause.

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Pull request merged**